Gemfile

```diff
@@ -2,6 +2,10 @@ source 'https://rubygems.org'
 
 gemspec
 
+platforms :ruby_19 do
+  gem 'json', '~> 1.8'
+end
+
```

Comment:
Why is this change needed? Oga doesn't even use the JSON Gem (at least directly) in any way.